devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -94,12 +96,20 @@ private getDOORLOCK_ATTR_MIN_PIN_LENGTH() { 0x0018 }
 private getDOORLOCK_ATTR_SEND_PIN_OTA() { 0x0032 }
 private getALARM_ATTR_ALARM_COUNT() { 0x0000 }
 private getALARM_CMD_ALARM() { 0x00 }
+private getIAS_ATTR_ZONE_STATUS() {0x0010}
 
 /**
  * Called on app installed
  */
 def installed() {
 	log.trace "ZigBee DTH - Executing installed() for device ${device.displayName}"
+
+	  if (device.getDataValue("model") == "E261-KR0B0Z0-HA") {
```

Comment:
Why are these in `installed` instead of `configure`?

Reply:
ok, it's moved to configure() method.